### PR TITLE
Dashboard header

### PR DESCRIPTION
**Pull request by mikepadiernos**
### Location:
- HeaderDashboard.js
- Header.js
- UserDashboard

### Details:
Separation of UserDashboard & PinnedCities initiated;
Header for secure view  refactored to present application UX 

### Commited Features:
updated logo & user menu 
initiated separation of userDashboard & PinnedCities

### CODE REVIEW

**Review by TheTrabin — COMMENTED** [review]
There's a few test errors with unused variables that should either be commented out or removed, but otherwise well build out.

Unsure if it was intended, but PinnedCities itself isn't called in App.js (App.js line 15 and App.js line 49?).

Markdown was called and commented out in List.stories.js, and it's tossing an unused error (List.stories.js Line 1 and Line 9).

**Comment by mikepadiernos** [comment]
I don't recall what the reason for the change in list stories but for the pinned cities I pushed another commit to another branch

File: src/components/common/LogoComponent.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cityspireLogo from '../../assets/imgs/cityspireLogoAlternative.png';
```
Comment (bobbyhalljr) [inline src/components/common/LogoComponent.js]: Please remove the unused variables. thanks!

**Review by bobbyhalljr — DISMISSED** [review]

File: src/components/common/LogoComponent.js (hunk 1 above)
Comment (mikepadiernos) [reply src/components/common/LogoComponent.js]: Done!

**Review by TheTrabin — APPROVED** [review]
No errors returning. Looks good

File: src/components/common/LogoComponent.js (hunk 1 above)
Comment (emfresh2o) [reply src/components/common/LogoComponent.js]: You really put an effort in creating a new logo, good job!

**Review by emfresh2o — APPROVED** [review]
I like how you were able to change the logo, good job!

**Pull request merged**